StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlCommentHelper.cs

```diff
@@ -171,6 +173,22 @@ internal static XmlNodeSyntax GetTopLevelElement(DocumentationCommentTriviaSynta
             return emptyElementSyntax;
         }
 
+        internal static IEnumerable<XmlNodeSyntax> GetTopLevelElements(DocumentationCommentTriviaSyntax syntax, string tagName)
+        {
+            var elements = syntax.Content.OfType<XmlElementSyntax>().Where(element => string.Equals(element.StartTag.Name.ToString(), tagName));
```

Comment:
:exclamation: I don't like that this returns all the `XmlElementSyntax` elements first, followed by the `XmlEmptyElementSyntax` elements. I would prefer the resulting list be in strict document order.